pkg/controller/upload-controller.go

```diff
@@ -361,6 +363,12 @@ func (r *UploadReconciler) createUploadPodForPvc(pvc *v1.PersistentVolumeClaim,
 		return nil, err
 	}
 
+	dv := &cdiv1.DataVolume{}
+	preallocation := false
+	if err := r.client.Get(context.TODO(), types.NamespacedName{Name: pvc.Name, Namespace: pvc.Namespace}, dv); err == nil {
```

Comment:
Same comment as in import controller, if you have an annotation you can write a utility and re-use it in all the controllers.